supernode/daemon/mgr/cdn/super_reader.go

```diff
@@ -171,3 +172,44 @@ func readTailer(ctx context.Context, reader io.Reader, pieceMd5 hash.Hash) error
 	}
 	return nil
 }
+
+func getMD5ByReadFile(reader io.Reader, pieceLen int32) (string, error) {
```

Comment:
Why not to invoke the function `readContent`? They're almost same.

Reply:
Done.